doc/index.md

```diff
@@ -0,0 +1,11 @@
+# Welcome to the FIWARE IoT Agent Framework
+
+This project aims to provide a Node.js module to enable IoT Agent developers to build custom agents for their devices that can
+easily connect to NGSI Context Brokers (such as [Orion](https://github.com/telefonicaid/fiware-orion) ). 
+
+An IoT Agent is a component that lets groups of devices send their data to and be managed from a FIWARE NGSI Context
+Broker using their own native protocols. IoT Agents should also be able to deal with security aspects of the Fiware
```

Comment:
Fiware -> FIWARE (as far as I remember, the later in the correct wording)

Reply:
Fixed in c0c6809

Reply:
That was the fix c0c6809

Reply:
As far as I see, c0c6809 fixed the ocurrence in README.md line 28, but this one is in doc/index.md line 7.

...or maybe github web is fooling me :)

Reply:
Ah `doc/index.md` - that explains it. To keep the PR small I was concentrating on enabling the Readthedocs build **only** - hence I only added the badge and the `yaml` file.  The documentation itself will need to be cleaned up  to be fully Readthedocs compliant (e.g the unnecessary anchor tags render poorly) - however this would make the PR much more noisy so I was going to tidy up in a separate PR.

Anyway, your criticism is valid for the file in question, and since this is a simple change I've now **fixed** it in 4f84890